configure_linux.yml

```diff
@@ -0,0 +1,25 @@
+- name: Configure splunk peernode linux tunables
+  hosts: peernode
+  user: root
+
+  tasks:
+    - name: add script blob to disable THP on boot
+      blockinfile:
```

Comment:
So, whenever I use `blockinfile`, I like to define a marker more obvious than the default "BEGIN/END", something like:
```
- name: disable THP on boot
  blockinfile:
    marker: "# {mark} ANSIBLE MANAGED BLOCK: disable-thp"
    block: whatever
```
**Note:** You have to edit the marker in the example to match the comment style for rc.local and splunk.conf...

This isn't required, but it makes it easier to see and avoid ansible-managed configuration when perusing config straight from disk.